site/cypress/e2e/courseMaterials.spec.js

```diff
@@ -185,7 +185,7 @@ describe('Test cases revolving around course material uploading and access contr
             cy.get('#submit-materials').click();
         });
 
-        cy.get('#cm-toggle-folders-btn').click();
+        cy.get('[onclick=\'folderopencookie();\']').click();
```

Comment:
Why is this change necessary?  It is better to target things by ID or by class, rather than by a bit of nasty HTML?  The same comment applies to the rest of the changes in this file.